src/index.html

```diff
@@ -1,13 +1,258 @@
+          >
+
+          <img
+            src="./img/section-category/category-ear-desktop.png"
+            alt="ears"
+            class="category__image"
+          >
+
+          <a href="#" class="category__header">All products</a>
+        </div>
+
+        <div class="category__categories">
+          <img
+            src="./img/section-category/category-ear-stick-one-desktop.png"
+            alt="one ear stick"
+            class="category__image"
+          >
+
+          <img
+            src="./img/section-category/category-ear-stick-desktop.png"
+            alt="ear stick"
+            class="category__image category__image--long"
+          >
+
+          <a href="#" class="category__header">Audio</a>
+        </div>
+
+        <div class="category__categories">
+          <img
+            src="./img/section-category/category-phone-desktop.png"
+            alt="phone"
+            class="category__image category__image--long"
+          >
+
+          <img
+            src="./img/section-category/category-charger-desktop.png"
+            alt="charger"
+            class="category__image"
+          >
+
+          <a href="#" class="category__header">Accessories</a>
+        </div>
+
+        <a href="#" class="button category__button">View all</a>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <div class="footer_background-image"></div>
+      <section class="about-us footer__about-us">
+        <h2 class="about-us__title section-title section-title--about-us">About us</h2>
+        <div class="about-us__text">
+          <p class="about-us__paragraph">
+            We’re a London-based tech company on a mission to remove barriers between people and technology.
+            <br><br>
+            To do this, we’re crafting intuitive, flawlessly connected products that improve our lives without getting in the way.
+            No confusing tech-speak. No silly product names. Just artistry, passion and trust. And products we’re proud to share with our friends and family. Simple.
+            <br><br>
+            Because beautiful tech shouldn’t be complicated.
+          </p>
+        </div>
+      </section>
+
+      <section class="contact-us footer__contact-us">
+        <h2 class="section-title contact-us__title">Contact us</h2>
+        <form
+          action="#"
+          class="contact-us__form"
+          onsubmit="this.reset(); return false;"
+        >
+          <input
+            type="text"
+            name="name"
+            placeholder="Name"
+            class="contact-us__input"
+            required
+          >
+
+          <input
+            type="email"
+            name="email"
+            placeholder="Email"
+            class="contact-us__input"
+            required
+          >
+
+          <textarea
+            name="message"
+            cols="30"
+            rows="4"
+            class="contact-us__input contact-us__message"
+          >Message</textarea>
```

Comment:
Message should be a placeholder, not the text
![image](https://github.com/mate-academy/layout_landing-page/assets/23013525/54578ec4-7807-4048-86d7-466ff517e61b)